src/middlewares/tripMiddleware.js

```diff
@@ -3,7 +3,14 @@ import { respondWithWarning } from '../helpers/responseHandler';
 import statusCode from '../helpers/statusCode';
 
 export const verifyTrip = async (req, res, next) => {
-  const trip = await findTripById(req.params.tripId);
+  
+  const id  = parseInt(req.params.tripId, 10) || parseInt(req.params.id, 10);
+  
+  if(!Number.isInteger(id)) {
+    return respondWithWarning(res, statusCode.resourceNotFound, 'Trip not found');
+  }
```

Comment:
please, these changes are not needed, your validation should take care of that. Thank you.